seml/config.py

```diff
@@ -413,29 +410,34 @@ def set_executable_and_working_dir(config_path, seml_dict):
     None
     """
     config_dir = str(Path(config_path).expanduser().resolve().parent)
-
     working_dir = config_dir
-    os.chdir(working_dir)
```

Comment:
Before existing and entering the context manager all the time, it makes more sense to have one wrapped around the main body of the function.

Reply:
As far as I understand, we change into different working directories in `determine_executable_and_working_dir` and thus cannot just wrap one context manager around the entire body.